src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
+};
+
+type BarStatProps = {
+  axes: AxisProp[];
+  legend: { title: string; subtitle: string };
+  position: string;
+};
+
+const BarStatCard = ({ axes, legend, position }: BarStatProps) => {
+  const isVertical = position === "vertical";
+  const statNumbers = axes.map((axis) => axis.y);
+  const maxNumber = Math.max(...statNumbers);
+  const fillPixels = statNumbers.map(
+    (number) => `${Math.round((number / maxNumber) * 100)}px`
```

Comment:
Sorry, I think I wasn't clear in my ticket, but I don't think these bars are actually meant to be scaled proportionally to the actual numbers. They're only meant to be viewed relatively or figuratively, so that you can see which one is the largest, but not necessarily proportional to the actual values.

We can just have the lengths of the bars be fixed at three static sizes, shortest, middle, and longest. This would simplify the implementation a lot, and it would also make the component more flexible in the cases where the actual "stat" might not be represented as a pure (JavaScript) number, for example, "700m" "800m" "1.2km".

Sorry for asking you to significantly rework this, but I think it'll at least be a lot more straightforward with fixed sizes. You may, however, want to add a prop that allows you to control whether the bars are in increasing or decreasing order, as a complement to the horizontal/vertical orientation prop you already have.